src/Main.java

```diff
@@ -1,5 +1,96 @@
 public class Main {
     public static void main(String[] args) {
 
+        // Задача 1
+
+        int age = 17;
+        if (age >= 18) {
+            System.out.println("Возраст человека равен 18, он совершеннолетний");
+
+        } else {
+            System.out.println("Ты не достиг совершеннолетия, нужно немного подождать");
+
+
+        }
+
+        // Задача 2
+
+        int temperature = 15;
+        if (temperature <= 5) {
+            System.out.println("На улице " + temperature + " градусов, нужно надеть шапку");
+
+        } else {
+            System.out.println("На улице " + temperature + " градусов, можно идти без шапки");
+        }
+
+        // Задача 3
+
+        int speed = 55;
+        if (speed >= 60) {
+            System.out.println("Скорость " + speed + ", придется заплатить штраф");
+        } else {
+            System.out.println("Скорость " + speed + ", можно ездить спокойно");
+        }
+
+        // Задача 4
+
+        int ageMan = 30;
+        if (ageMan >= 2 && ageMan <= 6) {
+            System.out.println("Возраст человека равен " + ageMan + " ему нужно ходить в детский сад");
+        }
+        else if (ageMan >= 7 && ageMan <= 17) {
+            System.out.println("Возраст человека равен " + ageMan + " ему нужно ходить в школу");
+        }
+        else if (ageMan >= 18 && ageMan <= 24) {
+            System.out.println("Возраст человека равен " + ageMan + " ему нужно ходить в университет");
+        } else  {
+            System.out.println("Возраст человека равен " + ageMan + " ему пора ходить на работу");
+
+        }
+
+        // Задача 5
+
+        int childAge = 15;
+        if (childAge < 5) {
+            System.out.println("Возраст ребенка равен " + childAge + ", ему нельзя кататься на аттракционе");
+        } else if (childAge > 5 && childAge <= 14) {
+            System.out.println("Возраст ребенка равен " + childAge + ", ему можно кататься на аттракционе" +
+                    " в сопровождении");
+        } else {
+            System.out.println("Возраст ребенка равен " + childAge + ", ему можно можно кататься на аттракционе " +
+                    "без сопровождения взрослого" );
+        }
+
+        // Задача 6
+
+        int wagon = 102;
+        int occupiedPlaces = 70;
+        int seatingPlaces = 60;
+        if (occupiedPlaces >= 60 && occupiedPlaces <= 102) {
+            System.out.println("Есть только стоячее место в вагоне");
+        } else if (occupiedPlaces < 60) {
```

Comment:
```suggestion
        } else if (occupiedPlaces < seatingPlaces) {
```
надо переменные использовать, а не числа